lib/ansible/modules/cloud/amazon/ecs_service.py

```diff
@@ -368,7 +376,8 @@ def create_service(self, service_name, cluster_name, task_definition, load_balan
             role=role,
             deploymentConfiguration=deployment_configuration,
             placementConstraints=placement_constraints,
-            placementStrategy=placement_strategy)
+            placementStrategy=placement_strategy,
+            launchType=launch_type)
```

Comment:
This doesn't seem to have any checks for whether boto3 supports `launchType`, which seems to only come with version 1.8.4 and beyond of botocore.

You'll need a check similar to https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/amazon/ecs_service.py#L443-L445

`launchType` should only be passed here if it's actually required.